webcompat/views.py

```diff
@@ -319,7 +320,20 @@ def create_issue():
                 url_for('show_issue', number=json_response.get('number')))
         # Authenticated reporting
         if form.get('submit_type') == 'github-auth-report':
-            if g.user:  # If you're already authed, submit the bug.
+            if g.user:
```

Comment:
This is to handle a case in the future when a user revokes a token, but is still logged in on webcompat.com and tries to submit an issue (not really required for this migration to GitHub apps, but nice to handle this exception).